packages/core/app/styles/navi-core/components/subtotal-dimension-select-trigger.less

```diff
@@ -4,6 +4,8 @@
 */
 
 .subtotal-dimension-select-trigger {
+  width: 100%;
+
```

Comment:
Do we still have/need `&__prefix` ?

Reply:
I could be wrong but I dont think we need this entire file anymore